layers/meta-balena-beaglebone/conf/layer.conf

```diff
@@ -37,14 +37,14 @@ KERNEL_IMAGETYPES:beaglebone-ai64 = "Image"
 KERNEL_CLASSES:beaglebone-ai64 = "kernel-uimage"
 
 PREFERRED_PROVIDER_virtual/kernel:beagleplay = "linux-bb.org"
-PREFERRED_PROVIDER_virtual/bootloader:beagleplay = "u-boot-ti-staging"
-PREFERRED_PROVIDER_u-boot:beagleplay = "u-boot-ti-staging"
+PREFERRED_PROVIDER_virtual/bootloader:beagleplay = "u-boot-bb.org"
```

Comment:
These changes can be removed from this commit and instead be made in the previous commit where the PREFERRED_PROVIDER for the bootloader is initially declared

Reply:
Agree. I think you have already made some of these changes (although they may have resulted in a regression). Will apply and update the PR.

Reply:
@delhiryder I did not make any changes to the PR. I only rebased it, the commits are all like you originally submitted them.

Reply:
Got it. Thanks

Reply:
Squashed commits as requested.